blockbuster/src/com/codecool/emis/blockbuster/Blockbuster.java

```diff
@@ -0,0 +1,97 @@
+package com.codecool.emis.blockbuster;
+
+import com.codecool.emis.blockbuster.product.Product;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class Blockbuster {
+
+    private Set<Product> products = new HashSet<>();
+    private Set<Product> blockBusters = new HashSet<>();
```

Comment:
it would be less memory consuming if you had only one set, and had a boolean blockBuster field in the Product class, that way you can still tell if a product is blockbusterfavorite or not, and you're using less memory because you're not storing the same information twice